### PR TITLE
RPRBLND-1836: conflict between the same naming in merged files

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Merge of two or more scenes with the same names in final and viewport render, shows only from the first scene 
Expected merge of the scenes get results from both in the final render

### EFFECT OF CHANGE
Added new Root node at Add -> USD -> Converters , that allows to create root primitive with specific name and type.

### TECHNICAL STEPS
Added and registered class RootNode
Added Root node to Add -> USD -> Converters menu
Added Name field
Added Type field

### CODE REVIEW

File: src/hdusd/usd_nodes/nodes/root.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        layout.prop(self, 'name')
+        layout.prop(self, 'type')
+
+    def compute(self, **kwargs):
+        input_stage = self.get_input_link('Input', **kwargs)
+        if not input_stage:
+            return None
+
+        if not self.name:
+            return None
```
Comment (bnagirniak) [inline src/hdusd/usd_nodes/nodes/root.py:58]: We can return input stage here

File: src/hdusd/usd_nodes/nodes/root.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+        name = Tf.MakeValidIdentifier(self.name)
+        stage = self.cached_stage.create()
+        UsdGeom.SetStageMetersPerUnit(stage, 1)
+        UsdGeom.SetStageUpAxis(stage, UsdGeom.Tokens.z)
+        root_prim = stage.GetPseudoRoot()
```
Comment (bnagirniak) [inline src/hdusd/usd_nodes/nodes/root.py:64]: this can be removed

File: src/hdusd/usd_nodes/nodes/root.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+        # create new root prim according to name and type
+        if self.type == 'Xform':
+            root_prim = UsdGeom.Xform.Define(stage, root_prim.GetPath().AppendChild(name))
```
Comment (bnagirniak) [inline src/hdusd/usd_nodes/nodes/root.py:68]: we can use just f'/{name}' here

**Review by bnagirniak — COMMENTED** [review]
Some code improvements

**Review by DagerD — APPROVED** [review]
Tested. Works ok.

**Review by bnagirniak — APPROVED** [review]

**Pull request merged**